pkg/operator/ceph/file/mds/livenessprobe.sh

```diff
@@ -7,7 +7,8 @@ FILESYSTEM_NAME="{{ .FilesystemName }}"
 KEYRING="{{ .Keyring }}"
 CMD_TIMEOUT="{{ .CmdTimeout }}"
 
-outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --keyring "$KEYRING" --connect-timeout="$CMD_TIMEOUT" --format json)"
+export CEPH_ARGS="--keyring $KEYRING" # workaround till ceph does not fix the issue with keyring
```

Comment:
Please reference the Ceph issue tracker number so we can know easily when it's safe to remove.


And to be honest, the text seems confusing, like it is saying Ceph isn't going to fix the issue. Just "workaround" with the issue link should suffice, like this

```suggestion
export CEPH_ARGS="--keyring $KEYRING" # workaround <link-to-ceph-tracker>
```

Reply:
Done